package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "tape-nock",
   "description": "Automatically record and playback HTTP calls for each tape test.",
-  "version": "1.2.2",
+  "version": "1.2.3",
```

Comment:
Ideally we should not bump the version as part of a pull request. There could be other incoming changes/PRs that get merged before this, so its usually best to do this separately as to avoid merge conflicts and allow for several changes to be grouped into a single publish.

Once this PR is accepted, we can do:
```bash
# edit the CHANGELOG.md and describe the changes, then:
npm version patch # or minor, or major, depending on the most impactful change
npm publish
git push origin master --tags # push the version bump commit and the tag back up to github
```